### PR TITLE
Done some more config for passport/jwt

### PR DESCRIPTION
**Pull request by Kysumi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+backend/dist/*
 dist/*
 backend/dist/*
```
Comment (jallier) [inline .gitignore]: We should probably only have this once

File: backend/src/config/passport.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import _ from "lodash";
+
+import { default as User } from "../db/entity/user.entity";
+import { Request, Response, NextFunction } from "express";
+
+import { getConnection, Repository } from 'typeorm';
+
+const LocalStrategy = passportLocal.Strategy;
+
+passport.serializeUser<any, any>((user, done) => {
```
Comment (jallier) [inline backend/src/config/passport.ts]: Is it possible to have types for these?

**Review by jallier — APPROVED** [review]
2 small tweaks, but they're pretty nitpicky. Looks great!

File: .gitignore (hunk 1 above)
Comment (Kysumi) [reply .gitignore]: whoops must have missed that when merging mah bad.

File: backend/src/config/passport.ts (hunk 2 above)
Comment (Kysumi) [reply backend/src/config/passport.ts]: Resolved.

**Pull request merged**